cmake/make_unit.cmake

```diff
@@ -10,19 +10,19 @@ include(add_target_parent)
 ##==================================================================================================
 # Unit test Configuration
 ##==================================================================================================
-add_library(unit_test_config INTERFACE)
+add_library(kiwaku_test INTERFACE)
 
-if (CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
-  target_compile_options( unit_test_config INTERFACE /W3 /EHsc /std:c++latest)
+if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
+  target_compile_options( kiwaku_test INTERFACE /W3 /EHsc /std:c++20)
 else()
-  if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-  target_compile_options( unit_test_config INTERFACE -std=c++20 -Werror -Wall -Wextra -Wno-gnu-string-literal-operator-template)
+  if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  target_compile_options( kiwaku_test INTERFACE -std=c++20 -Werror -Wall -Wextra -Wno-gnu-string-literal-operator-template)
```

Comment:
please silence this in the source...

Reply:
It may not be necessary anymore as raberu get updated for not using that anymore

Reply:
Removed as new rbr doesn't require this anymore